src/test_utils/mongo/mod.rs

```diff
@@ -1,13 +1,24 @@
-use std::str::FromStr;
-
+use crate::eth_provider::database::types::{
+    header::StoredHeader, receipt::StoredTransactionReceipt, transaction::StoredTransaction,
+};
 use crate::eth_provider::database::Database;
+#[cfg(any(test, feature = "arbitrary"))]
+use arbitrary::Arbitrary;
 use lazy_static::lazy_static;
+#[cfg(any(test, feature = "arbitrary"))]
+use mongodb::bson;
```

Comment:
Same here, I think you can group this with
```rust
#[cfg(any(test, feature = "arbitrary"))]
use {...}
```

Reply:
fixed